src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -277,11 +277,20 @@ public void showSecurityHotspotLocations(LiveSecurityHotspot liveSecurityHotspot
     showSecurityHotspot(liveSecurityHotspot, SecurityHotspotsPanel::selectLocationsTab);
   }
 
+  public boolean tryFindSecurityHotspot(String securityHotspotKey) {
```

Comment:
The name suggests a hotspot will be returned. Maybe reuse the `doesSecurityHotspotExist` name ?